src/Security/MemberAuthenticator/LostPasswordHandler.php

```diff
@@ -197,16 +189,55 @@ public function forgotPassword($data, $form)
 
         // Avoid information disclosure by displaying the same status,
         // regardless wether the email address actually exists
+        $identifier = isset($data[$uniqueIdentifier]) ? $data[$uniqueIdentifier] : '';
```

Comment:
Sorry I missed this one too. Also needs to be hard-coded to email. :)

Reply:
Ok, that is another part that would ideally be abstract. I'll push another commit that makes the use of `$data['Email']` more abstract, and see what you think

Reply:
@tractorcow let me know what you think of f3644484f8245c5089c04e799e0e000add466dc0